### ISSUE TITLE
Problems with a page view when /internals/* pages are accessed

### ISSUE THREAD

**Issue by BOLVERIN** [tempestphp/tempest-docs#24]
The /internals/* section of the website could be accessed without /docs/ in the URI, and the view is having issues.
![image_2024-09-28_130156380](https://github.com/user-attachments/assets/07477884-6467-41b3-97b5-010a39192b5d)

**Comment by brendt** [comment]
Yeah that's an old URL, it should be https://tempestphp.com/docs/internals/01-bootstrap/

I thought I updated all URLs pointing to this page, but maybe I missed one?

**Comment by BOLVERIN** [comment]
I found that old URL at the Brent's blog

**Comment by brendt** [comment]
A thanks! I forgot about those. It's fixed now :)
